src/order/conditionally_complete_lattice.lean

```diff
@@ -300,7 +300,6 @@ is larger than all elements of `s`, and that this is not the case of any `w<b`.
 See `Sup_eq_of_forall_le_of_forall_lt_exists_gt` for a version in complete lattices. -/
 theorem cSup_eq_of_forall_le_of_forall_lt_exists_gt (_ : s.nonempty)
   (_ : ∀a∈s, a ≤ b) (H : ∀w, w < b → (∃a∈s, w < a)) : Sup s = b :=
-have bdd_above s := ⟨b, by assumption⟩,
```

Comment:
This is clearly used. When I run the linter it points out line 353 of `order/complete_lattice`, where the equivalend `bdd_above s` assumption is in fact unused.

Reply:
Oops, I was too sloppy! Thanks

Reply:
I guess I just searched for the lemma name and one was a substring of the other!